src/app/main.ts

```diff
@@ -1,6 +1,7 @@
 import { app } from 'electron';
 import * as shellPath from 'shell-path';
 
+import * as electronDownloader from 'electron-dl';
```

Comment:
This should be grouped with the above import statements

Reply:
Good catch 👍 Fixed it.